src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -20,30 +20,109 @@
         class="mx-2"
         align="center"
       >
+        <!--        <v-col>-->
+        <!--          Get shareable link-->
+        <!--          <v-menu-->
+        <!--            offset-y-->
+        <!--            :close-on-content-click="false"-->
+        <!--            min-width="420"-->
+        <!--            max-width="420"-->
+        <!--          >-->
+        <!--            <template #activator="{ on }">-->
+        <!--              <v-icon-->
+        <!--                color="primary"-->
+        <!--                v-on="on"-->
+        <!--              >-->
+        <!--                mdi-link-->
+        <!--              </v-icon>-->
+        <!--            </template>-->
+        <!--            <v-card>-->
+        <!--              <CopyText-->
+        <!--                class="pa-2"-->
+        <!--                :text="permalink"-->
+        <!--                icon-hover-text="Copy permalink to clipboard"-->
+        <!--              />-->
+        <!--            </v-card>-->
+        <!--          </v-menu>-->
+        <!--        </v-col>-->
         <v-col>
-          Get shareable link
-          <v-menu
+          Share
+          <v-dialog
+            v-model="dialog"
             offset-y
-            :close-on-content-click="false"
             min-width="420"
-            max-width="420"
+            max-width="500"
           >
             <template #activator="{ on }">
               <v-icon
                 color="primary"
                 v-on="on"
               >
-                mdi-link
+                mdi-share
```

Comment:
```suggestion
                mdi-share-variant
```
https://materialdesignicons.com/icon/share-variant looks more like a social network share to me. `mdi-share` looks more like what I would call "export".